packages/app-project/src/helpers/fetchTranslations/fetchTranslations.spec.js

```diff
@@ -5,28 +5,23 @@ import fetchTranslations from './'
 describe('helpers > fetchTranslations', function () {
   before(function () {
     nock('https://panoptes-staging.zooniverse.org/api')
+    .persist()
     .get('/translations')
     .query(query => query.translated_type === 'project')
     .reply(200, {
       translations: [
         {
-          language: 'fr',
```

Comment:
I think I got this mock wrong. Without a language in the request query, the API should only return the project default language (I think.)

Reply:
I could be wrong. When the query doesn’t specify a language, the API might send back all the languages for a resource. Either way, I’d expect the response to include `en`.